lib/rails_admin/config.rb

```diff
@@ -86,6 +86,9 @@ class << self
       attr_accessor :navigation_static_links
       attr_accessor :navigation_static_label
 
+      # Set where RailsAdmin fetches JS/CSS from, defaults to :sprockets
```

Comment:
We need to ensure compatibility with existing applications. Since using `:webpacker` as default value for `asset_source` doesn't automatically make those applications Webpacker-ready, we need to provide migration path for users.

In this way, users can proceed like this:
- Upgrade RailsAdmin, keeping the app functional since RailsAdmin delivers assets using Sprockets by default
- Make their app Webpacker-ready
- When things are ready, they can configure RailsAdmin to use Webpacker-sourced assets